batavia/core/builtins.js

```diff
@@ -35,6 +35,23 @@ batavia.builtins.__import__ = function(args, kwargs) {
     if (module === undefined) {
         module = batavia.modules.sys.modules[args[0]];
     }
+    // Check if there is a stdlib (pyc) module.
+    if (module === undefined) {
```

Comment:
I'm pretty sure I agree with this ordering; but sanity checking to make sure it was intentional, not accidental:

This means that a module in modules namespace will be loaded before the stdlib, which means a module written in Javascript will supersede one provided as Python bytecode. So, for example, the decimal module, which was originally written in Python, could be replaced with a native Javascript version by just dropping it into the sys.modules namespace.